_data-prepper/pipelines/configuration/processors/date.md

```diff
@@ -23,8 +23,13 @@ locale | No | String | Locale is used for parsing dates. It's commonly used for
 
 <!---## Configuration
 
-Content will be added to this section.
+Content will be added to this section.--->
 
 ## Metrics
 
-Content will be added to this section.--->
\ No newline at end of file
+<!--- Add the following? "Apart from common metrics in [AbstractProcessor](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-api/src/main/java/org/opensearch/dataprepper/model/processor/AbstractProcessor.java)..."--->
+
+The date processor introduces the following custom metrics:
+
+* `dateProcessingMatchSuccessCounter`: Returns the number of records that match with at least one pattern specified in the match configuration option. <!--- Should "match configuration option/patterns match configuration option" be in quotes, or tic marks? --->
```

Comment:
@graytaylor0 Should "match configuration option/patterns match configuration option" be in quotes or inside tic marks?